ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.0__4.7.yaml

```diff
@@ -5,10 +5,8 @@ build_root:
     tag: rhel-8-golang-1.19-openshift-4.12
 images:
 - dockerfile_path: Dockerfile
-  from: src
```

Comment:
I would undo these changes to non master branch files

I believe the problem does not happen in those branches

Reply:
As discussed offline removing this line won't break, so we will have this in sync between different builders/branches.